assets/scss/components/_rvtd-notifications-menu.scss

```diff
@@ -64,6 +64,18 @@
                             0 0 0 4px $color-alert-green;
             }
         }
+
+        &[aria-expanded="true"]>span>svg.rvt-closed-icon {
```

Comment:
Let's make sure to put an empty space between each element in a chained selector like these here. So these would become:

```sass
&[aria-expanded="true"] > span > svg.rvt-closed-icon {
  // property declarations
}
```